File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
    
    boolean twice_check = false;

    int counter = 0;

    for(int i = 0; i < nums.length; i++) {

        int m = nums[i];

        for(int z = 0; z < nums.length; z++) {

            if (m == nums[z]){
                counter++;
            }

            if(counter >= 2){
                twice_check = true;
            }
        }
        counter = 0;
    }


    return twice_check;

    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        
        res = 0
        cnt = 0
        for num in nums:
            if num:
                cnt = cnt + 1 
            else:
                cnt = 0
            
            if cnt > res:
                res = cnt
        return res